cogs/database_bg.py

```diff
@@ -0,0 +1,43 @@
+from datetime import datetime
+from discord.ext import tasks,commands
+from coc import utils
+
+class Database_Background(commands.Cog):
+    def __init__(self, bot):
+        self.bot = bot
+        self.update.start()
+
+    @commands.command(name="add_user")
+    async def add_user(self, ctx, player_tag):
+        "Command is used to register a user to the database"
+        player_tag = utils.correct_tag(player_tag)
+        player = await self.bot.coc.get_player(player_tag)
+        commit = self.bot.dbconn.register_user((
+            player.tag,
+            player.name,
+            player.town_hall,
+        ))
+        if commit:
+            await ctx.send(f'Error: {commit}')
+        else:
+            await ctx.send(f'Adding user {player.name}')
+        
+    @tasks.loop(minutes=3.0)
+    async def update(self):
+        "This method updates the database every 3 mintues"
+        tags = self.bot.dbconn.get_players()
+        tag_list = [ tag[0] for tag in tags ]
```

Comment:
This could be an iter - that way we're not storing them in memory

```suggestion
        tag_iter = (n[0] for n in tag_data)
```